### DISCUSSION TITLE
Gopher support in w3m

### DISCUSSION THREAD

**Discussion by Poorchop** [Homebrew/discussions#959] [Everyday usage]
Is it possible to compile w3m with support for the Gopher protocol?

It also seems like inline image support is disabled but I'm not understanding why especially since terminals like iTerm2 have support for inline images. Is image support only available exclusively in X?

**Comment by SMillerDev** [comment]
> Is it possible to compile w3m with support for the Gopher protocol?

Probably, you should check the upstream documentation for this.

> It also seems like inline image support is disabled but I'm not understanding why especially since terminals like iTerm2 have support for inline images. Is image support only available exclusively in X?

What is required to support this will also be documented upstream.

**Reply by Poorchop** [reply]
I was more asking if the formula could be modified to do this. If not, then I'll fork and try it myself.

**Reply by SMillerDev** [reply]
Probably, you'll have to try for yourself and make a pull request.
